### DISCUSSION TITLE
Grabación 30 de agosto

### DISCUSSION THREAD

**Discussion by pjdiaz9731** [Launch-X-Latam/MisionFrontEnd#122] [General]
Hola alguno sabe si el live de ayer sera subido a grabaciones,ya que no pude asistir y realmente me preocupa lo visto.

**Comment by samujm** [comment]
Puedes verlo aqui

https://teams.microsoft.com/l/meetup-join/19%3ameeting_OGU2ZmY2ODAtZjc2Zi00YTk4LTk2Y2QtMWUxMmM0NjQ0ZjBh%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d

Solo que creo copia el link en tu navegador por que la verdad creo git no te redirecciona

**Reply by samujm** [reply]
Cuando tengas ese problema que no aparecen las grabaciones, unicamente dale clic a la liga que puso Sherpa y te manda al video grabado.

**Comment by pjdiaz9731** [comment]
Muchas gracias por la aclaración !!
